scenarios/aws/ec2/vm_run.go

```diff
@@ -112,6 +116,10 @@ func VMRunWithDocker(ctx *pulumi.Context) error {
 				fakeIntakeOptions = append(fakeIntakeOptions, fakeintake.WithLoadBalancer())
 			}
 
+			if env.AgentUseDualShipping() {
```

Comment:
Agent Dual shipping was an option available only on Kubernetes. If we add that, passing the flag `ddagent:dualshipping` would only deactivate the fakeintake payload forwarding because agent dual shipping option is not offered on non-kubernetes agent. Which would be misleading because we end up in a situation where nothing is sent to dddev anymore